java/client/src/main/java/glide/api/models/commands/LPosOptions.java

```diff
@@ -0,0 +1,53 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.models.commands;
+
+import java.util.ArrayList;
+import java.util.List;
+import lombok.Builder;
+
+/**
+ * Optional arguments to {@link glide.api.commands.ListBaseCommands#lpos(java.lang.String,
+ * java.lang.String, glide.api.models.commands.LPosOptions)} and {@link
```

Comment:
Please add imports and shorten the link
```suggestion
 * Optional arguments to {@link ListBaseCommands#lpos(String, String, LPosOptions)} and ...
```